src/main/java/core/mate/academy/service/TruckProducerImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducerImpl implements MachineProducer {
+    @Override
+    public List<Machine> get() {

```

Comment:
Consider changing the return type of the `get` method to `List<Truck>` for better type safety and clarity, as the method currently returns a list of `Truck` instances.